docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -0,0 +1,9 @@
+---
+id: installation
```

Comment:
💭 Wondering, would this be a good place to also describe the default state? i.e. if you don't do anything then you'll have:
1. Basic auth
2. Unprotected API
3. A`demo:demo` user
4. Authorizations disabled
5. An admin role